changelogs/fragments/1363-update-docstring-system.yml

```diff
@@ -0,0 +1,3 @@
+trivial:
+  - system - Updated docstrings to numpy style for visual aid to developers.
+    (https://github.com/ansible-collections/ibm_zos_core/pull/1336).
```

Comment:
you swapped two characters: should be pull 1363, not 1336

Reply:
you right